Gemfile

```diff
@@ -41,16 +41,16 @@ gem 'omniauth-cas'
 gem 'omniauth-saml'
 
 # Honeycomb
-gem 'sequel'
-gem 'honeycomb-beeline'
-gem 'rack-honeycomb', '~> 0.5.0'
+#gem 'sequel'
+#gem 'rack-honeycomb', '~> 0.5.0'
+gem 'honeycomb-beeline', '>= 1.0.0'
 
 # Recaptcha
 gem "recaptcha", require: "recaptcha/rails"
 
-group :production do
-  gem 'clamav', git: 'https://github.com/eagleas/clamav', branch: 'master'
-end
+#group :production do
```

Comment:
If we're cutting clamav, it would be nice to see it as a separate issue with some reasoning behind it for documentation reasons.

Reply:
we can cut clamav in a separate PR but clamav is currently broken and fails builds